package.json

```diff
@@ -48,17 +48,14 @@
     "babel-runtime": "6.26.0",
     "cross-env": "3.2.4",
     "css-loader": "0.28.11",
-    "del": "2.2.2",
-    "deterministic-tar": "0.2.0",
```

Comment:
We're using `deterministic-tar` for Linux.

Reply:
I don't find any occurance in code. We use `tar-stream`.

Reply:
Ah, my mistake.